scss/_tables.scss

```diff
@@ -4,6 +4,7 @@
 
 .table {
   --#{$variable-prefix}table-bg: #{$table-bg};
+  --#{$variable-prefix}table-accent-bg: #{$table-bg};
```

Comment:
I don't know if the problem comes from this modification (haven't studied it), but there are some problems in the Tables documentation in this PR :

- https://deploy-preview-680--boosted.netlify.app/docs/5.0/content/tables/
- https://boosted.orange.com/docs/5.0/content/tables/

Reply:
That was indeed what was messing with the displaying of rows. Thanks for the heads-up !